coordinator/userapi.go

```diff
@@ -122,6 +137,8 @@ func (s *userAPIServer) SetManifest(ctx context.Context, req *userapi.SetManifes
 		return nil, status.Errorf(codes.Internal, "setting manifest: %v", err)
 	}
 
+	s.metrics.manifestGeneration.Inc()
```

Comment:
```suggestion
	s.metrics.manifestGeneration.Set(len(s.manifSetGetter.GetManifests()))
```
Does not make a difference right now, but will when we support recovery.